frameworks/geniverse/resources/animation_core.js

```diff
@@ -84,6 +85,7 @@ sc_require('lib/burst-core');
         timeline,
         swapui,
         frame = 0,
+        fps = 10,
```

Comment:
This `fps` is a tricky one to choose. I agree that it's wise to make it slow (like you have here) so it doesn't eat cycles out of the main animation. However, at least on my machine, I get a weird "breadcrumbs trail" effect when it is this slow. Parts of the chromosomes get left behind as the front of the chromosomes move. For me, at least, the chromosome movement looks much smoother at 20 fps, while not affecting the overall animation speed at all.